pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -399,7 +405,7 @@ func (am *Alertmanager) applyConfig(cfg *apimodels.PostableUserConfig, rawConfig
 }
 
 func (am *Alertmanager) WorkingDirPath() string {
-	return filepath.Join(am.Settings.DataPath, workingDir)
+	return filepath.Join(am.Settings.DataPath, workingDir, strconv.Itoa(mainOrgID))
```

Comment:
The path under which the templates and silences will be stored will eventually be:
[paths.data configuration](https://grafana.com/docs/grafana/latest/administration/configuration/?plcmt=footer#data)/alerting/<org_id>
eg: `data/alerting/1`.
Is that what we want?

Reply:
That seems correct to me.